pkg/apis/k0s/v1beta1/cplb.go

```diff
@@ -34,9 +34,34 @@ type ControlPlaneLoadBalancingSpec struct {
 	// +optional
 	Enabled bool `json:"enabled,omitempty"`
 
+	// type indicates the type of the node-local load balancer to deploy on
+	// worker nodes. Currently, the only supported type is "Keepalived".
+	// +kubebuilder:default=Keepalived
+	// +optional
+	Type CPLBType `json:"type,omitempty"`
```

Comment:
Do we really need the type at all since we have only one?

Reply:
This makes the API future-proof. Maybe there's alternatives to keepalived. Having the type here makes adding new stuff easier. Folks don't have to specify it in the YAMLs since Keepalived is the default. We do similar things for NLLB.

Reply:
👍

Reply:
Since we're all in the same page I'm marking this as resolved